contracts/DelFiPrice.sol

```diff
@@ -22,37 +27,22 @@ contract DelFiPrice is OpenOracleView {
      * @param messages The messages to post to the oracle
      * @param signatures The signatures for the corresponding messages
      */
-    function postPrices(bytes[] calldata messages, bytes[] calldata signatures) external {
+    function postPrices(bytes[] calldata messages, bytes[] calldata signatures, string[] calldata symbols) external {
         require(messages.length == signatures.length, "messages and signatures must be 1:1");
 
-        // Initialize the list of unique symbols
-        string[] memory symbols = new string[](messages.length);
-        uint numSymbols = 0;
-
         // Post the messages, whatever they are
         for (uint i = 0; i < messages.length; i++) {
-            string memory symbol = OpenOraclePriceData(address(data)).put(messages[i], signatures[i]);
-
-            // Possibly add to the unique symbol set
-            bool found = false;
-            for (uint j = 0; j < symbols.length; j++) {
-                if (keccak256(abi.encodePacked(symbol)) == keccak256(abi.encodePacked(symbols[j]))) {
-                    found = true;
-                    break;
-                }
-            }
-            if (!found) {
-                symbols[i] = symbol;
-                numSymbols += 1;
-            }
+            OpenOraclePriceData(address(data)).put(messages[i], signatures[i]);
```

Comment:
I notice that OpenOraclePriceData#put still returns the key / symbol. Do you want to keep that since it isn't being used anymore?

Reply:
Ya intentional - I think its still useful in general or else callers need to know how to decode to extract the key that was written.